boost-maven/boost-boms/boost-mp20-apis-bom/pom.xml

```diff
@@ -0,0 +1,119 @@
+<!-- Copyright (c) 2018 IBM Corporation and others. All rights reserved. 
+	This program and the accompanying materials are made available under the 
+	terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+	and is available at http://www.eclipse.org/legal/epl-v10.html Contributors: 
+	IBM Corporation - initial API and implementation -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.boost</groupId>
+    <artifactId>boost-mp20-apis-bom</artifactId>
+    <!--  Versioned separately from the main project -->
+    <version>0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+	<description>Booster MP20 APIs Bom</description>
+
+    <properties>
```

Comment:
Maybe this could include a link to the MP aggregrate POM you took this from?  Or at least the GAV coordinates.